package.json

```
@@ -1386,7 +1386,7 @@
     "coverage": "npm test -- --coverage",
     "compile-tests": "del-cli ./assets/test/**/.build && npm run compile",
     "package": "vsce package",
-    "dev-package": "vsce package --no-update-package-json 1.12.0-dev",
+    "dev-package": "vsce package --no-update-package-json 2.0.0-dev",
```

This needs to be higher than the released version
```suggestion
    "dev-package": "vsce package --no-update-package-json 2.0.1-dev",
```

---

Good point. Thanks!